draft#6

alimak86 opened the pull request:
изменения:
 - поиск контакта по почте
 - поиск контакта по имени
 - исправлено поле вводы месяца

Commit:
draft#6

изменения:
 - поиск контакта по почте
 - поиск контакта по имени
 - исправлено поле вводы месяца

Petrenko-Anton reviewed (changes requested):
пошук повинен бути з одного поля пошуку, не важливо, це ім'я, прізвище або email

Commit:
draft#7

- работа с комментариями для контактов корректан
 - поиск реализован по имени, фамилии и почте (логический опрератор или)
- поправлена шаблон contacts, потому что необходимо явно указывать, метод изменения формы GET/POST. Если переходить к обработчику только лишь по ссылке, то система не понимает, какой запрос и в этом случае форма становится инвалидной. Надо преукрасить кнопку поиска.
- переменные среды для погоды и валют фиктивные, олько лишь для запуска сервера (должны быть поменяны)

Petrenko-Anton reviewed (approved):
ОК
